tests/util.py

```diff
@@ -75,3 +77,26 @@ def set_env(**environ):
     finally:
         os.environ.clear()
         os.environ.update(old_environ)
+
+
+def inject_sitecustomize(path):
```

Comment:
@SeanOC avoids duplication, making this utility function helpful for other modules. Even if it can be considered generic, I've added the wording `sitecustomize` to narrow the usage scope of the function.